test/e2e/multicast.go

```diff
@@ -188,7 +188,7 @@ func testMulticastIGMPQuery(f *framework.Framework, clientNodeInfo, serverNodeIn
 	createGenericPod(f, multicastListenerPod, serverNodeInfo.name, f.Namespace.Name, tcpDumpCommand)
 
 	// Wait for tcpdump on listener pod to be ready
-	err := wait.PollImmediate(retryInterval, retryTimeout, func() (bool, error) {
+	err := wait.PollUntilContextTimeout(context.Background(), retryInterval, retryTimeout, true, func(context.Context) (bool, error) {
```

Comment:
Could you add a comment to the boolean ? `true /*immediate*/`

Although maybe is better to use ginkg.Eventually here since it integrate with test framework.

Reply:
My intent here is just to stop using the deprecated api. Hope you are okay with me adding `true /*immediate*/` to minimize my change on this pr.